roud-permissions.ttl

```diff
@@ -36,10 +36,10 @@
 #
 
 ### Default Object Access Permission on Admin
-<http://rdfh.ch/permissions/0112/roud-oeuvres-o001> a knora-base:DefaultObjectAccessPermission ;
-	knora-base:hasPermissions "CR knora-base:ProjectAdmin|D knora-base:ProjectMember" ;
-	knora-base:forProject <http://rdfh.ch/projects/0112> ;
-	knora-base:forGroup knora-base:ProjectAdmin .
+#<http://rdfh.ch/permissions/0112/roud-oeuvres-o001> a knora-base:DefaultObjectAccessPermission ;

```

Comment:
I realize now that the `DefaultObjectAccessPermission` on `ProjectAdmin` and `ProjectMember` groups were ill-formed: they both have the same IRI. There must be things to cleanup in the permissions graph.